framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/database/external/ExternalDatabaseProvider.java

```diff
@@ -27,13 +27,15 @@
     private static Collection<ExternalDatabase> availableExternalDatabases = new ArrayList<>();
 
     static {
-        availableExternalDatabases.add(new Db2ExternalDatabase());
```

Comment:
This file can be removed (for now) as it is replaced by TemplateExternalDatabaseProvider. Just please adjust also KieServerWithExternalDatabaseScenarioImpl to reference it.